testing/endtoend/evaluators/operations.go

```diff
@@ -87,7 +87,7 @@ var ValidatorsHaveExited = e2etypes.Evaluator{
 // SubmitWithdrawal sends a withdrawal from a previously exited validator.
 var SubmitWithdrawal = e2etypes.Evaluator{
 	Name:       "submit_withdrawal_epoch_%d",
-	Policy:     policies.BetweenEpochs(helpers.CapellaE2EForkEpoch-2, helpers.CapellaE2EForkEpoch+1),
```

Comment:
@nisdas does this seem reasonable? rather than make a new policy func that submits withdrawals 2 epochs before capella *if* capella's epoch >= 2 and otherwise submits at capella (0) i'm just setting this up to start running at capella. I'm not sure if I need to also update `ValidatorsHaveWithdrawn`.